design/FY2021/NFP-Spaces.md

```diff
@@ -0,0 +1,234 @@
+
+Enclosures are used for radiant and solar/daylighting exchange. Enclosures are primarily concerned with Surfaces, but they
+are also related to internal gains (which cast radiant and visible energy into the enclosure). By default there is one Enclosure for each
+Zone.  By using Construction:AirBoundary, multiple zones may be grouped into a single larger Enclosure. 
+Enclosures are assigned automatically based on the zones connected by an air boundary surface. 
+
+
+## New Space Object
+
+### Relationship assumptions ##
+
+If we keep the current definition of "Zone" and add the concept of
+"Space" (<= Zone), then this would add a new layer to the data model:
+
+    Zone
+        Thermostat
+        HVAC equipment
+        Space 2
+            Surfaces
+            Internal Gains (People, Lights, etc.)
+        Space 3
+            Surfaces
+            Internal Gains (People, Lights, etc.)
+
+    Enclosure
+        Space 1
+            Surfaces
+            Internal Gains (People, Lights, etc.)
+        Space 2
+            Surfaces
+            Internal Gains (People, Lights, etc.)
+    
+
+### Definitions
+
+ * Surface - A geometric plane which is attached to a Space. 
+   * A Surface can be opaque, transparent, or an air boundary.
+   * Each Surface belongs to one Space.
+
+ * Space - A collection of one or more Surfaces and internal gains.
+   * Each Space belongs to one Zone (explicitly user-assigned).
+   * The Spaces in a Zone are lumped together for the Zone heat balance.
+   * There is no heat balance at the Space level.
+   * Each Space belongs to one Enclosure (implicitly assigned).
+
+ * Zone - An air mass connecting Surfaces, internal gains, and HVAC equipment for heat balance and HVAC control.
+   * Each Zone is comprised of one or more Spaces.
+   * All Surfaces and internal gains associated with the Spaces are included in the Zone.
+   * The Zone heat balance does not change: it has an air node (or a Room Air Model) and includes all Surfaces and 
+   internal gains from its Spaces plus any HVAC which is attached to the Zone.
+
+ * Enclosure - A continuous volume connecting Surfaces for radiant, solar, and daylighting exchange.
+   * Each Enclosure is comprised of one or more Spaces.
+   * There is one Enclosure for each Space unless there are air boundary surfaces 
+   which group multiple Spaces into a single Enclosure.
+   * All Surfaces and internal radiant gains associated with the Spaces are included in the Enclosure.
+   * Enclosures only distribute radiant (thermal), solar, and visible energy to and from the Surfaces.
+   * There is no full heat balance at the Enclosure level. Each Enclosure only balances the radiant/solar flux on each 
+   Surface. These fluxes then become part of the Surface inside heat balance.
+
+## Proposed Input Approach ##
+Very minimal changes to current inputs.
+
+ * Old Zone object becomes Space object plus some tags.
+ * New Zone object has a name and a list of Spaces.
```

Comment:
Suggest the name ThermalZone which would align with OpenStudio and allow for other types of Zones.  It sounds like Enclosure is kind of like a RadiantHeatTransferZone.

Reply:
It seems like these different *Zones could be used to partition the simulation into different calculations and might even be useful in parallelizing the simulation in the future?